src/webextension/firstrun/index.js

```diff
@@ -7,6 +7,9 @@ import ReactDOM from "react-dom";
 
 import AppLocalizationProvider from "../l10n";
 import App from "./components/app";
+import * as telemetry from "../telemetry";
+
+telemetry.recordEvent("lockbox", "render", "firstrun");
```

Comment:
I'm not sure if this is the right place to put this (but I'm not sure it's wrong either); if it were me, I'd put it in `componentDidMount()` for the `<App/>` component, but the two should be roughly equivalent. The former would fire even if we screwed something up and the page failed to render, whereas the latter wouldn't fire. Just something to think about... vOv